src/Core/src/Platform/iOS/SemanticExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Foundation;
+using UIKit;
+using System;
+
+namespace Microsoft.Maui
+{
+	public static partial class SemanticExtensions
+	{
+		public static void SetSemanticFocus(IFrameworkElement element)
+		{
+			if (element?.Handler == null)
+				throw new NullReferenceException("Can't access view from a null handler");
```

Comment:
This will appear if I set focus on say a ctor. That is technically valid and can be fixed later, but we could maybe have a "pending" property and set when the handler is created.

Reply:
Yea, pending is just a really hard concept with announce because if you want to queue up an announce then you have to interrupt whatever the OS naturally wants to announce. In most (maybe all?) cases that would be considered bad practice. 

So we're being hostile here intentionally for now :-)